internal/services/mssql/mssql_server_transparent_data_encryption_resource.go

```diff
@@ -45,11 +45,18 @@ func resourceMsSqlTransparentDataEncryption() *pluginsdk.Resource {
 				ForceNew:     true,
 				ValidateFunc: mssqlValidate.ServerID,
 			},
+
 			"key_vault_key_id": {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
 				ValidateFunc: keyVaultValidate.NestedItemId,
 			},
+
+			"auto_rotation_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Computed: true,
```

Comment:
out of interest why make this computed? this'll have a default value of `false`, which'll be returned via `d.Get` below so we may as well use that as the default here and set it via a state migration?

Reply:
Sure! I've never done a state migration, but I can definitely a try!